app/lib/guidelines/Guidelines.js

```diff
@@ -471,7 +471,7 @@ export default [
             }
         },
         severity: SEVERITY.WARNING,
-        link: 'https://github.com/Noel-Bastubbe/for-Construction-Modeling/wiki/Dependency-Model#dep2---check-if-all-objectives-are-reachable-from-the-start-state'
+        link: 'https://github.com/Noel-Bastubbe/for-Construction-Modeling/wiki/Dependency-Model#dep2---link-every-objectives-to-at-least-one-other-objective'

```

Comment:
bitte durch
https://github.com/Noel-Bastubbe/for-Construction-Modeling/wiki/Dependency-Model#dep2---link-every-objective-to-at-least-one-other-objective
ersetzen (objectives zu objective geändert)